docs/releases/1.17-NOTES.md

```diff
@@ -72,6 +72,8 @@ the notes prior to the release).
  
 * Support for CoreOS has been deprecated and will be removed in future versions of Kops. Those affected should consider using [Flatcar](../operations/images.md#flatcar) as a replacement.
 
+* Support for the "Legacy" etcd provider has been deprecated. It will not be supported for Kubernetes 1.18 or later. 
```

Comment:
Maybe link to instructions on how to move off of the legacy etcd provider - https://kops.sigs.k8s.io/etcd3-migration/